.github/workflows/test-runner-on-perf.yml

```diff
@@ -62,6 +65,56 @@ jobs:
           done
+      - name: "Assemble data-repo code to generate version.properties file"
+        run: ./gradlew clean assemble
+      - name: "Read latest jade-data-repo gitHash from version.properties file"
+        id: read_property
+        run: |
+          VERSIONPROPERTIESPATH=./src/main/resources/version.properties
+          VERSIONPROPERTY=gitHash
+          VERSIONRESULT=$(sed -n "/^[[:space:]]*$VERSIONPROPERTY[[:space:]]*=[[:space:]]*/s/^[[:space:]]*$VERSIONPROPERTY[[:space:]]*=[[:space:]]*//p" "$VERSIONPROPERTIESPATH")
+          echo "property value: $VERSIONRESULT"
+          echo ::set-output name=value::"$VERSIONRESULT"
+      - name: 'Checkout datarepo-helm-definitions repo'
+        uses: actions/checkout@v2
+        with:
+          repository: 'broadinstitute/datarepo-helm-definitions'
+          token: ${{ secrets.HELM_REPO_TOKEN }}
+          path: datarepo-helm-definitions
+      - name: "Update perf image tag with gitHash"
+        uses: docker://mikefarah/yq:latest
+        with:
+          args: yq w -i datarepo-helm-definitions/perf/datarepo-api.yaml image.tag ${{ steps.read_property.outputs.value }}
+      - name: "Create datarepo-helm-definition pull request with updated perf image tag"
+        uses: broadinstitute/create-pull-request@v3 # forked from peter-evans/create-pull-request
+        id: perfTagPR
+        with:
+          token: ${{ secrets.HELM_REPO_TOKEN }}
+          path: datarepo-helm-definitions
+          commit-message: "Perf Datarepo version update: ${{ steps.read_property.outputs.value }}"
+          committer: datarepo-bot <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          title: "Perf Datarepo version update: ${{ steps.read_property.outputs.value }}"
+          branch: "version-update-${{ steps.read_property.outputs.value }}"
+          body: |
+            Update versions in perf env to reflect image tag ${{ steps.apiprevioustag.outputs.tag }}.
+            *Note: This PR was opened by the [update-env GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
+          labels: "broadbot,datarepo,automerge"
+      - name: "Merge datarepo-helm-definition pull request"
+        run: |
+          cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-helm-definitions
+          git checkout master
+          git config pull.rebase false
+          if git rev-parse --verify origin/version-update-${{ steps.read_property.outputs.value }}; then
+            git merge origin/version-update-${{ steps.read_property.outputs.value }}
+            git branch -d version-update-${{ steps.read_property.outputs.value }}
+            git push
+          else
+            echo "Branch version-update-${{ steps.read_property.outputs.value }} not found."
+          fi
+          cd ${GITHUB_WORKSPACE}/${workingDir}
+      - name: "Setup helmfile"
+        uses: broadinstitute/setup-helmfile@v0.5.0
```

Comment:
This step updates the version of the perf env helm chart in the helmfile. Is that right? And could you point me to where this helmfile lives?

Reply:
This step just installs helmfile onto the VM. 
But, great catch -> I had moved some things around and had deleted the step that actually does the "helmfile apply." Glad to answer more questions if this doesn't clear things up.

Reply:
Got it, thanks.